sections/04_study.md

```diff
@@ -188,139 +188,91 @@ disease, which will lead to novel diagnostics and therapeutics.
+generative methods by using k-mer features [@tag:Ghandi2014_enhanced
+@tag:Setty2015_seqgl], but string kernel based SVM systems are limited by
+expensive computational cost proportional to the number of training and testing
+sequences. 
+
+With the advent of deep learning, Alipanahi et al.
 [@tag:Alipanahi2015_predicting] showed that convolutional neural network models
 could achieve state of the art results on the TFBS task and are scalable to a
-large number of genomic sequences.
-
-Since the work by [@tag:Alipanahi2015_predicting] was published, there have been
-a multitude of deep learning works on the TFBS task. Due to the  motif-driven
-nature of the TFBS task, most architectures have been convolutional-based, as
-summarized in [@tag:Zeng2016_convolutional]. [@tag:Lanchantin2016_motif]
-introduced several new convolutional and  recurrent neural network models for
-predicting TFBSs, which showed improvements over other deep learning models on
-the dataset from [@tag:Alipanahi2015_predicting].  While many models for TFBS
-prediction resemble computer vision and natural language processing (NLP) tasks,
-it is important to note that DNA sequence tasks are fundamentally different than
-NLP tasks, and thus the models should be adapted from traditional deep learning
-models in order to account for such differences. For example, motifs may appear
-in either strand of a DNA sequence, resulting in two different forms of the
-motif (forward and reverse complement) due to complementary base pairing. To
-handle this issue, [@tag:Shrikumar2017_reversecomplement] created a
-convolutional model which can find motifs in both directions. Since deep
-learning for protein binding prediction is still in early stages, we expect to
-see an increase in domain-specific architectures for this task.
-
-Deep learning models have shown great accuracy on TFBS prediction, but the
-results are not fully convincing for several reasons. First, ChIP-seq
-experiments give a continuous value of binding likelihood at a certain location
-based on many experiments at that location in DNA. Based on these values, the
-TFBS task is usually converted into a binary classification task based on a
-certain threshold of the ChIP-seq value. However, the ChIP-seq values are often
-noisy, resulting in a target class that may be incorrect dependent on the
-defined threshold.  Converting the task into a binary classification isn't
-completely accurate since the model may give a high probability of a binding
-site, but the ChIP-seq binding signal may be just barely over the binary
-classification threshold, resulting in potential false positive (or the signal
-may be just below the threshold, resulting in potential false negatives).
-
-Second, most datasets for TFBS prediction are separated by TF, requiring a
-separate model for each TF (i.e. binary classification on each TF sub-dataset).
-In reality, there may be multiple TFs binding at the same location and TF
```

Comment:
@agitter Yes, I meant because they stack the final classifier on top. They do predict the individual ChIP-seq tracks for specific TFs in a specific cell type, but they report the TFBS, DHS, and HM AUC scores compared to previous techniques, so I think that it's ok to put it here.

I think I talked too much about how most models use TFs separately, which is true, but the data for all TFs at once is there from ENCODE (which is what they use in DeepSEA), so I think it's good to just mention DeepSEA here and show that they did TF binding prediction the right way.